Wrappers/Python/test/test_algorithms.py

```diff
@@ -1289,28 +1244,22 @@ def test_SPDHG_vs_PDHG_explicit(self):
         sigma = 1/normK
         tau = 1/normK
 
-        f1 = alpha * MixedL21Norm()
+        f1 = MixedL21Norm()
         f = BlockFunction(f1, f2)
         # Setup and run the PDHG algorithm
         pdhg = PDHG(f=f, g=g, operator=operator, tau=tau, sigma=sigma)
-        pdhg.max_iteration = 1000
         pdhg.update_objective_interval = 200
-        pdhg.run(1000, verbose=0)
-
-        # %% show diff between PDHG and SPDHG
-        # plt.imshow(spdhg.get_output().as_array() -pdhg.get_output().as_array())
-        # plt.colorbar()
-        # plt.show()
+        pdhg.run(100)
 
         qm = (mae(spdhg.get_output(), pdhg.get_output()),
               mse(spdhg.get_output(), pdhg.get_output()),
               psnr(spdhg.get_output(), pdhg.get_output())
               )
         log.info("Quality measures: %r", qm)
         np.testing.assert_almost_equal(mae(spdhg.get_output(), pdhg.get_output()),
-                                       0.00150, decimal=3)
+                                       0.0031962995, decimal=3)
```

Comment:
Where do these numbers come from? Are we expecting the error to be close to zero?

Reply:
see above